ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/90888_Automation_AdHoc_Smoke_Screen-Size/90893_SmokeTestSetUpExport.js

```diff
@@ -1,38 +0,0 @@
-/// <reference types ='Cypress'/>
-// Guide: The exported smoke test will run within the screeSizeImport.js File.
-// Step1: To use this effectively include your test commands into this file without the 'Before' & 'After' Hooks.
-// Step2: Ensure to include notes for each test so that changes can be updated easily as needed for each smoke run. Example is provided below.
-
-export const smokeTest = () => {
```

Comment:
Why are these tests being removed?

Reply:
This was a spike for applying screen size's and applying them to a subset of tests which are outdated and no longer in need.